samples/XCT.Sample/Pages/Effects/TouchEffectPage.xaml

```diff
@@ -17,6 +17,17 @@
         <StackLayout Padding="{StaticResource ContentPadding}"
                      Spacing="10">
 
+            <StackLayout
+                Orientation="Horizontal">
+                <Label
+                    HorizontalOptions="FillAndExpand"
+                    FontSize="Title"
+                    TextColor="Black"
```

Comment:
Default TextColor for Label is already black

Reply:
@nichojo89 No, default TextColor is Color.Default